readme.md

```diff
@@ -228,6 +228,7 @@ Thanks for contributing! 🦋🙌
 - [](# "update-pr-from-base-branch") [Adds button to update a PR from the base branch to ensure it builds correctly before merging the PR itself.](https://user-images.githubusercontent.com/1402241/57941992-f2170080-7902-11e9-8f8a-594aad983559.png) GitHub only adds it when the base branch is "protected".
 - [](# "quick-review-buttons") [Simplifies the PR review form: Approve or reject reviews faster with one-click review-type buttons.](https://user-images.githubusercontent.com/1402241/34326942-529cb7c0-e8f3-11e7-9bee-98b667e18a90.png)
 - [](# "warn-pr-from-master") [Warns you when creating a pull request from the default branch, as it’s an anti-pattern.](https://user-images.githubusercontent.com/1402241/52543516-3ca94e00-2de5-11e9-9f80-ff8f9fe8bdc4.png)
+- [](# "separate-draft-pr-button") [Separates `Create draft pull request` and `Create pull request` buttons.](https://user-images.githubusercontent.com/202916/66609826-4a081980-ebba-11e9-80d8-cdef503a8a80.png)
```

Comment:
I think this should go up one, instead of splitting the two that start with Warns.

Reply:
They’re not in a particular order. Let’s just spend the features to each list instead of adding them in the middle.

Reply:
I get that they're not in the order, but it seems wrong to disrupt the flow that's already there. Appending will solve this too.